code/modules/antagonists/cult/cult_comms.dm

```diff
@@ -151,6 +151,11 @@
 	var/datum/antagonist/cult/antag = owner.mind.has_antag_datum(/datum/antagonist/cult,TRUE)
 	if(!antag)
 		return
+	var/datum/objective/eldergod/summon_objective = locate() in antag.cult_team.objectives
+	var/area/place = get_area(owner)
+	if(place in summon_objective?.summon_spots)
```

Comment:
Wait, why are you comparing this against every single summonable area? Just get the turfs used by the ritual and filter those out

Reply:
Wouldn't that allow the leader to loop around this by summoning their cultists just a few moments right before the ritual rune is finished on those turfs though? Or am I misunderstanding? Also I'm not sure how to go about with that, just was pasting this from a dead PR.

Reply:
areas contain hundreds of turns, this would ban say, all of medbay, or all of science

Reply:
Any area where you could summon

Reply:
What would stop you from just starting the ritual prior to starting the narsie rune